frontend/src/i18n/messages/en/index.ts

```diff
@@ -438,17 +438,18 @@ export const messages = {
   },
   Header: {
     navLinks: {
+      about: "About",
       home: "Home",
-      roadmap: "Roadmap",
-      research: "Research",
-      vision: "Vision",
-      subscribe: "Subscribe",
-      menuToggle: "Menu",
-      search: "Search",
       login: "Sign in",
       logout: "Sign out",
+      menuToggle: "Menu",
+      research: "Research",
+      roadmap: "Product Roadmap",
```

Comment:
The pattern we're following even with page titles is to only capitalize the first word, so we should use `Product roadmap` and `Our vision` here

Reply:
Sounds good! Also, I was asked to split the refactor into a separate PR. 
My draft PR: https://github.com/HHS/simpler-grants-gov/pull/4639 includes the solution for nav dropdown state management. It was built off of this one, so it also includes this work!